### PR TITLE
Update elm to version 0.19

### LINKED ISSUE #1953: Update elm to v0.19
Updated version of the elm example already available [here](https://github.com/evancz/elm-todomvc)

### PR DESCRIPTION
**Pull request by Razzeee**
Here's an update for the elm version.

Fixes #1953

### CODE REVIEW

File: examples/elm/index.html [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
-	<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
-	<style>
-		body {
-			width: auto;
-		}
-		.todomvc-wrapper {
-			visibility: visible !important;
-		}
-	</style>
+  <meta charset="UTF-8">
```
Comment (passy) [inline examples/elm/index.html]: Could you retain the tabs? Some more rules are outlined here: https://github.com/tastejs/todomvc/blob/master/codestyle.md

File: examples/elm/index.html [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
+    <link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+    <style>
+      body {
+        width: auto;
+      }
+      .todomvc-wrapper {
+        visibility: visible !important;
+      }
+    </style>
+  <script type="text/javascript" src="build/elm.js"></script>
```
Comment (passy) [inline examples/elm/index.html]: You can drop the `type` here.

File: examples/elm/index.html [hunk 3]
```diff
@@ -1,34 +1,31 @@
...
-		var savedModel = result ? JSON.parse(result) : null;
-
-		var todomvc = Elm.Todo.fullscreen(savedModel);
+</body>
 
-		todomvc.ports.save.subscribe(function (model) {
-			localStorage.setItem('elm-todo-model', JSON.stringify(model));
-		});
-	}());
+<script type="text/javascript">
```
Comment (passy) [inline examples/elm/index.html]: Can you wrap the code in a closure here to not leak the variables to the global scope?

**Comment by FadySamirSadek** [comment]
solves #1954

File: examples/elm/index.html (hunk 3 above)
Comment (Razzeee) [reply examples/elm/index.html]: Can you show how you want me to do this?

**Pull request merged**